File: VB/VGridControl/SomeList.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace E4423
    Public Class SomeList
        Public Sub New()
        End Sub
        Public Sub New(ByVal _number As Integer, ByVal _name As String, ByVal _something As String)
            Number = _number
            Name = _name
            Something = _something

        End Sub
        Public Property Number() As Integer
        Public Property Name() As String
        Public Property Something() As String
    End Class
End Namespace

File: CS/VGridControl/SomeList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace E4423
{
    public class SomeList
    {
        public SomeList() { }
        public SomeList(int _number, string _name, string _something)
        {
            Number = _number;
            Name = _name;
            Something = _something;
        
        }
        public int Number { get; set; }
        public string Name { get; set; }
        public string Something { get; set; }
    }
}
